src/triton_fil/model_instance_state.cuh

```diff
@@ -50,15 +50,17 @@ class ModelInstanceState : public BackendModelInstance {
       TritonTensor<const float>& data, TritonTensor<float>& preds,
       bool predict_proba = false);
 
-  raft::handle_t& get_raft_handle();
+  raft::handle_t* get_raft_handle();
 
   ModelInstanceState(
       ModelState* model_state,
       TRITONBACKEND_ModelInstance* triton_model_instance, const char* name,
-      const TRITONSERVER_InstanceGroupKind kind, const int32_t device_id);
+      const int32_t device_id);
 
  private:
   ModelState* model_state_;
+  void* treelite_handle_;  // non-owning reference to the Treelite object
```

Comment:
Any reason not to read this off `model_state_` directly when needed?

I'm trying to think if there would ever be a time when this would end up pointing to the wrong spot if we cache it as an attribute of the instance state, and I'm not sure.